nengo/tests/test_connection.py

```diff
@@ -613,6 +613,45 @@ def test_slicing_function(Simulator, plt, seed):
     assert np.allclose(w, y, atol=0.1)
 
 
+def test_advanced_indexing(Simulator, plt, seed):
+    N = 100
+
+    model = nengo.Network()
```

Comment:
You should pass the seed into the Network (`Network(seed=seed)`) so that the test becomes deterministic; otherwise, we'll get random parameters each time and potentially random test failures! :smile: